### ISSUE TITLE
Quantize RequestMemory

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#8460]
This one is a bit trickier because:
1) we need to make sure the MaxRSS follows the same RequestMemory settings
2) we should quantize at the very top level, such that it makes it to the request spec to avoid misleading information

I think 250MB quanta would suffice, no?
And of course, get rid of of the HTCondor one.

**Comment by vlimant** [comment]
can you make the MaxRSS in the worker node always aligned with RequestMemory that the jobs sees from htcondor ?

**Comment by bbockelm** [comment]
Can we loop SI into this ticket?  There's a few places where rounding can occur -- we want to make sure they are all lined up.

@jamesletts can you flag this and bring in whoever else needs to take a look?  (GitHub auto-complete doesn't suggest anyone else...)

**Comment by aperezca** [comment]
Hi, yes, I'm watching these issues as well.
In the last SI meeting we agreed to drop any rounding (disk, time and also memory, anything else?) in one of the WMAgent schedds and rely on quantization coming from upstream. We'd need to check that number of autoclusters and negotiation time for that schedd remains under control, before extending to the whole set of agents.

**Comment by amaltaro** [comment]
Antonio, the RelVal agent has the newest version with Disk/Walltime quantization in place. The node is vocms0259. Can you have a look to make sure everything is correct?

There are more nodes going into production today, see https://cms-logbook.cern.ch/elog/GlideInWMS/6058

**Comment by bbockelm** [comment]
Is this done?

**Comment by amaltaro** [comment]
Nope, RequestMemory requires more changes such that we make everything consistent starting from the request creation. Feel free to suggest a milestone.
